docs/configuration.md

```diff
@@ -157,6 +157,18 @@ Defaults to one of the following directories:
 - Windows: `C:\Users\<username>\AppData\Local\pypoetry\Cache`
 - Unix:    `~/.cache/pypoetry`
 
+## `experimental.system-git-client`
+
+**Type**: boolean
+
```

Comment:
Should we add:
```suggestion

*Introduced in 1.2.0*

```
like we have [here](https://github.com/python-poetry/poetry/blob/c68011d8e5e3172181ea65c556b66a393c09e17c/docs/configuration.md?plain=1#L186)?

Reply:
Done